var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -42,6 +42,7 @@ class Hdf5(Package):
     version('1.8.13', 'c03426e9e77d7766944654280b467289')
 
     variant('debug', default=False, description='Builds a debug version of the library')
+    variant('shared', default=False, description='Builds a static executable version of the library')
```

Comment:
Yes, I need to update this.  I'll make the changes so that this package enables the "shared" variant by default and so that the descriptions are correct.